vignettes/site_only/db_connections.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+title: "Database Connections using healthcare.ai"
+author: "Mike Mastanduno"
+date: "4/1/2018"
+output: html_document
+---
+
+**Database connections**
+
+Let's be honest. You need to get data from a database, do some really awesome 
+stuff with `R`, and then write the results back to a database. `R` can help
+with that. There are lots of different databases, but this document will focus
+on Microsoft SQL Server. R supports most databases, and 
+I'd encourage you to look at their [help pages](https://db.rstudio.com/) 
+for additional resources. Andiamo!
+
+If you simply want a code snippet to copy and paste, scroll to the bottom.
+
+# Reading data
+
+#### Connecting to a Database
+
+Before you can do anything, you must make a connection to your database. The 
+`build_connection_string` function takes care of the syntax for you and returns
+a connection string that's ready to be passed into `DBI::dbConnect`. After
+you have a valid connection, you should be able to browse the database using 
+RStudio's Connections pane. If you don't have a connections pane, upgrade your 
+Rstudio. It's worth it.
+
+```{r, eval=FALSE}
+library(healthcareai)
+library(DBI)
+
+my_con <- build_connection_string(server = "HCS-GM0004",
+                                  database = "SAM")
+con <- DBI::dbConnect(odbc::odbc(), .connection_string = my_con)
```

Comment:
Either attach the package with library or specify the namespace with ::, but not both.

Reply:
Removed.